packages/font/scripts/update-google-fonts.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs/promises')
+const path = require('path')
+
+;(async () => {
+  const { familyMetadataList } = await fetch(
```

Comment:
The Node.js version used by the CI might not have `globalThis.fetch` available. Maybe the existing `devDependencies` `node-fetch` should be imported?

Reply:
Thanks!